scene/main/node.cpp

```diff
@@ -809,6 +809,14 @@ bool Node::is_processing_internal() const {
 	return data.idle_process_internal;
 }
 
+void Node::set_process_priority(int p_priority) {
```

Comment:
I would check that the node is in that group before forcing it to be changes, you can check with is_processing, is physics_processing, etc. and so on.